.ci/test-local-tap.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+set -eo pipefail
+
+. "${0%/*}/libfunctions.sh"
+
+echo '--- :evergreen_tree: Configuring Environment'
+exec 9>&1 # enable tee to write to STDOUT as a file
+test_bottle
+
+echo '+++ :beer: Homebrew Tap and Install'
+perform 'brew update'
+if [[ ! $(python3 --version 2>/dev/null) ]]; then
+   perform 'brew install python3'
+fi
+perform "sed -i.bk -e 's/url \".*\"/url \"http:\/\/127.0.0.1:7800\"/' ${BUILDKITE_PIPELINE_SLUG}/*.rb"
+perform "pushd ${BUILDKITE_PIPELINE_SLUG} && git add *.rb && git commit -m 'test it!' && popd"
```

Comment:
**Optional, UX** - Same as with the package test steps in [eos](https://github.com/EOSIO/eos), I think the ampersands here are cluttered and pointless. It would be more readable on separate lines, IMHO, and it would be easier to determine which of these commands printed an error, if there is one, saving time debugging.

Reply:
I like the idea of keeping things that require directory changes (which are a side-effect) together. Since this fits on a single line without wrapping, I think its not unreasonable.